sunpy/coordinates/transformations.py

```diff
@@ -66,6 +68,28 @@
            'hme_to_gei', 'gei_to_hme', 'gei_to_gei']
 
 
+# Boolean flag for whether to ignore the inertial motion of the Sun
+_ignore_sun_motion = False
+
+
+@contextmanager
+def ignore_sun_motion():
+    """
+    Context manager for ignoring the inertial motion of the Sun for transformations.
+
+    Essentially, coordinates will be assumed to move along with the Sun over time (e.g., for
+    features on the Sun itself).
```

Comment:
It should be clear here that this doesn't take into account any rotation or differential rotation of the surface of the Sun.